dependencies.xml

```diff
@@ -53,10 +53,7 @@ Note all install methods after "main" take
     <pyside2/>
     <nomkl os='linux' skip_check='True'/>
     <numexpr os='linux'/>
-    <ray os="mac,linux" source="pip">1.3</ray>
-    <!-- ray is not compatible with aioredis 2.0
-      https://github.com/ray-project/ray/issues/17475 -->
-    <aioredis os="mac,linux" source="pip">1.3</aioredis>
+    <ray os="mac,linux" source="pip"/>
```

Comment:
Do we want to switch to an unversioned ray?

Reply:
1.8 sounds fine to me.

Reply:
FYI, on our Mac test machine, ray 1.8 is not available. On my local mac machine, ray 1.8 can be found and installed. In this case, I will fix the ray version to 1.5. @joshua-cogliati-inl 
```
... Activating environment ...
... Installing libraries from PIP-ONLY ...
...pip-only command: pip install ray==1.8.* xmlschema
ERROR: Could not find a version that satisfies the requirement ray==1.8.* (from versions: 0.6.0, 0.6.1, 0.6.2, 0.6.3, 0.6.4, 0.6.5, 0.6.6, 0.7.0, 0.7.1, 0.7.2, 0.7.3, 0.7.4, 0.7.5, 0.7.6, 0.7.7, 0.8.0, 0.8.1, 0.8.2, 0.8.3, 0.8.4, 0.8.5, 0.8.6, 0.8.7, 1.0.0rc0, 1.0.0rc1, 1.0.0rc2, 1.0.0, 1.0.1, 1.0.1.post1, 1.1.0, 1.2.0, 1.3.0, 1.4.0rc1, 1.4.0rc2, 1.4.0, 1.4.1, 1.5.0, 1.5.1, 1.5.2)
ERROR: No matching distribution found for ray==1.8.*

ERROR: exiting with code 1
```